tests/src/Interop/COM/NativeServer/StringTesting.h

```diff
@@ -225,8 +225,11 @@ class StringTesting : public UnknownImpl, public IStringTesting
         /*[in]*/ LPWSTR a,
         /*[out]*/ LPWSTR b)
     {
-        ReverseInplace(::wcslen(b), b);
-        return S_OK;
+        // Not possible to test from native server
```

Comment:
Couldn't we also use the `UnmanagedType.VBByRefStr` marshalling behavior to correctly test this from the native server?

Reply:
Not really. This is testing the out attribute only. Changing the marshaling type is a different test. The point here is we can't properly test the out attribute on a default wide-string marshal.